src/Domain/BlockNeighbor.hpp

```diff
@@ -18,18 +18,21 @@ class er;
 /// \endcond
 
 /// \ingroup ComputationalDomainGroup
-/// Information about the neighbor of a Block in a particular direction.
+/// Information about the neighbor of a host Block in a particular direction.
 ///
 /// \tparam VolumeDim the volume dimension.
 template <size_t VolumeDim>
 class BlockNeighbor {
  public:
   BlockNeighbor() = default;
 
-  /// Construct with the Id and orientation of the neighbor.
+  /// Construct with the Id and orientation of the neighbor relative to the
+  /// host.
   ///
   /// \param id the Id of the neighbor.
-  /// \param orientation the OrientationMap of the neighbor.
+  /// \param orientation the OrientationMap of the neighbors relative to the
+  /// host. In other words, the OrientationMaps accept objects in the host
```

Comment:
This OrientationMap takes objects in the host Block and returns what they are in the neighbor Block.

Reply:
@kidder how about adding "... what they are in the coordinates of the neighbor Block"? I find this phrasing makes it clearer that the same object is viewed in new coordinates.

Reply:
@fmahebert  "... takes objects in the logical coordinate frame of the host Block and maps them to the logical coordinate frame of the neighbor Block" ?

Reply:
Sounds good to me